components/content-service/pkg/logs/logs.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package logs
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	// TerminalStoreLocation is the path in the workspace where terminal related data like logs or hist-files are
+	// persisted in the file system
+	TerminalStoreLocation = "/workspace/.gitpod"
+
+	prebuildLogFilePrefix = "prebuild-log-"
+
+	legacyTerminalStoreLocation = "/workspace"
+	legacyPrebuildLogFilePrefix = ".prebuild-log-"
```

Comment:
We've made the switch from this "legacy" naming convention to what we use today about nine months ago - I reckon we could get rid of it.

Not necessarily as part of this PR, only if it makes you life/this code easier.